duckduckgo/iOS
# Feature/surrogates

Reviewer: Mia or Caine
Asana: https://app.asana.com/0/414235014887631/520419619790473
CC:

**Description**:

Uses surrogates when tracking scripts are blocked which break page functionality.

Note that raspberrypi.org is whitelisted so can't be used for testing, but the ones below are valid tests even with the whitelist.

**Steps to test this PR**:
1. Visit https://www.xxl.fi/urheilu-ja-pallopelit/pelit/lelut-ja-pelit/c/161404 - select an item and view the details
1. Visit https://www.trovaprezzi.it/ - site works
1. Visit https://www.eff.org/files/pbtest/ga_js_surrogate_test.html - the message 'it works!' should appear 
1. Visit other sites and confirm they still work
